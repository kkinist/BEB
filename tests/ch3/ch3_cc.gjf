%chk=ch3.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc

BEB step 6: CCSD(T) for neutral ch3

0 2

C H 0
cc-pVTZ
****

