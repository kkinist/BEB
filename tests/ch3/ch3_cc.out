 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-27851.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     27853.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch3.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc
 ---------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 -----------------------------------
 BEB step 6: CCSD(T) for neutral ch3
 -----------------------------------
 Structure from the checkpoint file:  "ch3.chk"
 Charge =  0 Multiplicity = 2
 Redundant internal coordinates found in file.
 C,0,0.,-0.0000000011,0.
 H,0,0.000000002,-1.0827517802,0.
 H,0,-0.9376905476,0.5413758867,0.
 H,0,0.9376905457,0.5413758901,0.
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000   -1.082752    0.000000
      3          1           0       -0.937691    0.541376    0.000000
      4          1           0        0.937691    0.541376    0.000000
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4
     1  C    0.000000
     2  H    1.082752   0.000000
     3  H    1.082752   1.875381   0.000000
     4  H    1.082752   1.875381   1.875381   0.000000
 Stoichiometry    CH3(2)
 Framework group  D3H[O(C),3C2(H)]
 Deg. of freedom     1
 Full point group                 D3H     NOp  12
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    1.082752    0.000000
      3          1           0       -0.937691   -0.541376    0.000000
      4          1           0        0.937691   -0.541376    0.000000
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    285.1563071    285.1563071    142.5781536
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.8236000000D+04  0.5419783203D-03
      0.1235000000D+04  0.4192873817D-02
      0.2808000000D+03  0.2152216205D-01
      0.7927000000D+02  0.8353432195D-01
      0.2559000000D+02  0.2395828457D+00
      0.8997000000D+01  0.4428528419D+00
      0.3319000000D+01  0.3517995618D+00
 S   6 1.00       0.000000000000
      0.2808000000D+03 -0.5949224937D-04
      0.7927000000D+02 -0.1148158310D-02
      0.2559000000D+02 -0.1001913745D-01
      0.8997000000D+01 -0.6121949230D-01
      0.3319000000D+01 -0.1732698541D+00
      0.3643000000D+00  0.1072915192D+01
 S   1 1.00       0.000000000000
      0.9059000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1285000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1871000000D+02  0.3942638716D-01
      0.4133000000D+01  0.2440889849D+00
      0.1200000000D+01  0.8154920089D+00
 P   1 1.00       0.000000000000
      0.3827000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1209000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1097000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.3180000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.7610000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      4 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    35 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     8 symmetry adapted cartesian basis functions of A2  symmetry.
 There are    22 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    15 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    30 symmetry adapted basis functions of A1  symmetry.
 There are     8 symmetry adapted basis functions of A2  symmetry.
 There are    20 symmetry adapted basis functions of B1  symmetry.
 There are    14 symmetry adapted basis functions of B2  symmetry.
    72 basis functions,   103 primitive gaussians,    80 cartesian basis functions
     5 alpha electrons        4 beta electrons
       nuclear repulsion energy         9.6437168407 Hartrees.
 NAtoms=    4 NActive=    4 NUniq=    2 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    72 RedAO= T EigKep=  2.77D-03  NBF=    30     8    20    14
 NBsUse=    72 1.00D-06 EigRej= -1.00D+00 NBFU=    30     8    20    14
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (?A) (?A) (?B) (?B) (?C)
       Virtual   (?A) (?B) (?B) (?B) (?B) (?C) (?A) (?B) (?B) (?A)
                 (?D) (?D) (?A) (?A) (?B) (?B) (?C) (?D) (?D) (?B)
                 (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B) (?B)
                 (?D) (?D) (?A) (?B) (?B) (?C) (?A) (?D) (?D) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?C) (?A)
                 (?B) (?B) (?D) (?D) (?C) (?A) (?B) (?B) (?B) (?B)
                 (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Beta  Orbitals:
       Occupied  (?A) (?A) (?B) (?B)
       Virtual   (?C) (?A) (?B) (?B) (?B) (?B) (?C) (?A) (?B) (?B)
                 (?A) (?D) (?D) (?A) (?A) (?B) (?B) (?C) (?D) (?D)
                 (?B) (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?C) (?A) (?D) (?D)
                 (?C) (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?C) (?A) (?B) (?B) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 0.5000 <S**2>= 0.7500 S= 0.5000
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=5456056.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -39.5773216377     A.U. after   11 cycles
            NFock= 11  Conv=0.39D-08     -V/T= 2.0017
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 0.5000 <S**2>= 0.7619 S= 0.5059
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     0.7619,   after     0.7501
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    72
 NBasis=    72 NAE=     5 NBE=     4 NFC=     1 NFV=     0
 NROrb=     71 NOA=     4 NOB=     3 NVA=    67 NVB=    68

 **** Warning!!: The largest alpha MO coefficient is  0.11833050D+02


 **** Warning!!: The largest beta MO coefficient is  0.11829360D+02

 Semi-Direct transformation.
 ModeAB=           4 MOrb=             4 LenV=    1048253326
 LASXX=       174447 LTotXX=      174447 LenRXX=      358619
 LTotAB=      184172 MaxLAS=      685008 LenRXY=           0
 NonZer=      533066 LenScr=     1571840 LnRSAI=      685008
 LnScr1=     1571840 LExtra=           0 Total=      4187307
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=1 Pass  1:  I=   1 to   4.
 (rs|ai) integrals will be sorted in core.
 ModeAB=           4 MOrb=             3 LenV=    1048253326
 LASXX=       134806 LTotXX=      134806 LenRXX=      261658
 LTotAB=      126852 MaxLAS=      513756 LenRXY=           0
 NonZer=      396464 LenScr=     1188864 LnRSAI=      513756
 LnScr1=     1541120 LExtra=           0 Total=      3505398
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=2 Pass  1:  I=   1 to   3.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.7570458788D-02 E2=     -0.2232937156D-01
     alpha-beta  T2 =       0.3853857272D-01 E2=     -0.1283763116D+00
     beta-beta   T2 =       0.2497560235D-02 E2=     -0.7520971137D-02
 ANorm=    0.1024014937D+01
 E2 =    -0.1582266543D+00 EUMP2 =    -0.39735548291953D+02
 (S**2,0)=  0.76188D+00           (S**2,1)=  0.75315D+00
 E(PUHF)=      -0.39580511471D+02        E(PMP2)=      -0.39737569937D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=5403492.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 E(PMP3)=      -0.39754843370D+02
 MP4(R+Q)=  0.30304229D-02
 E3=       -0.18139071D-01        EUMP3=      -0.39753687362D+02
 E4(DQ)=   -0.13059104D-02        UMP4(DQ)=   -0.39754993273D+02
 E4(SDQ)=  -0.20093982D-02        UMP4(SDQ)=  -0.39755696761D+02
 DE(Corr)= -0.17330427     E(Corr)=     -39.750625907    
 NORM(A)=   0.10307196D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17748529     E(CORR)=     -39.754806930     Delta=-4.18D-03
 NORM(A)=   0.10327512D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17852850     E(CORR)=     -39.755850133     Delta=-1.04D-03
 NORM(A)=   0.10332637D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17883885     E(CORR)=     -39.756160493     Delta=-3.10D-04
 NORM(A)=   0.10333981D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17888249     E(CORR)=     -39.756204124     Delta=-4.36D-05
 NORM(A)=   0.10334087D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17888456     E(CORR)=     -39.756206199     Delta=-2.08D-06
 NORM(A)=   0.10334110D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17888503     E(CORR)=     -39.756206670     Delta=-4.71D-07
 NORM(A)=   0.10334117D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17888527     E(CORR)=     -39.756206905     Delta=-2.34D-07
 NORM(A)=   0.10334117D+01
 Iteration Nr.   9
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.17888523     E(CORR)=     -39.756206871     Delta= 3.38D-08
 NORM(A)=   0.10334116D+01
 Largest amplitude= 3.66D-02
 Time for triples=        4.07 seconds.
 T4(CCSD)= -0.48388872D-02
 T5(CCSD)=  0.99874522D-04
 CCSD(T)= -0.39760945884D+02

 S**2, projected HF & approx projected MPn energies after annihilation of
 unwanted spin states (see manual for definitions):

 spins       (S**2,0) (S**2,1)    PUHF        PMP2        PMP3        PMP4
 annihilated
 s+1         0.74996  0.75001  -39.580511  -39.737570  -39.754843
 s+1,s+2     0.75000  0.75000  -39.580503  -39.737562  -39.754836
 s+1 to s+3  0.75000  0.75000  -39.580503  -39.737562  -39.754836

 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (?A) (?A) (?B) (?B) (?C)
       Virtual   (?A) (?B) (?B) (?B) (?B) (?C) (?A) (?A) (?B) (?B)
                 (?D) (?D) (?A) (?A) (?B) (?B) (?C) (?D) (?D) (?B)
                 (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B) (?B)
                 (?D) (?D) (?A) (?C) (?A) (?B) (?B) (?D) (?D) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?A) (?C)
                 (?B) (?B) (?D) (?D) (?C) (?B) (?B) (?A) (?B) (?B)
                 (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Beta  Orbitals:
       Occupied  (?A) (?A) (?B) (?B)
       Virtual   (?C) (?A) (?B) (?B) (?B) (?B) (?A) (?C) (?A) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?C) (?D) (?D)
                 (?B) (?B) (?B) (?B) (?A) (?A) (?C) (?B) (?B) (?B)
                 (?B) (?D) (?D) (?A) (?C) (?B) (?B) (?A) (?D) (?D)
                 (?C) (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?A)
                 (?C) (?B) (?B) (?D) (?D) (?C) (?B) (?B) (?A) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Unable to determine electronic state:  an orbital has unidentified symmetry.
 Alpha  occ. eigenvalues --  -11.23454  -0.94575  -0.57858  -0.57858  -0.38477
 Alpha virt. eigenvalues --    0.14957   0.22187   0.22187   0.37705   0.37705
 Alpha virt. eigenvalues --    0.44068   0.48793   0.61995   0.66318   0.66318
 Alpha virt. eigenvalues --    0.73110   0.73110   1.02825   1.06694   1.09034
 Alpha virt. eigenvalues --    1.09034   1.15652   1.26721   1.26721   1.44273
 Alpha virt. eigenvalues --    1.44273   1.54433   1.54433   1.81064   2.36413
 Alpha virt. eigenvalues --    2.39030   2.54102   2.54102   2.84163   2.84163
 Alpha virt. eigenvalues --    2.90583   2.90583   3.16259   3.19183   3.21799
 Alpha virt. eigenvalues --    3.23055   3.23055   3.27569   3.27569   3.49595
 Alpha virt. eigenvalues --    3.53615   3.68185   3.68185   3.91219   3.91219
 Alpha virt. eigenvalues --    3.95153   3.95153   4.16402   4.20813   4.21939
 Alpha virt. eigenvalues --    4.27519   4.27519   4.39955   4.39955   4.99087
 Alpha virt. eigenvalues --    5.16845   5.16845   5.19522   5.21077   5.21077
 Alpha virt. eigenvalues --    5.28780   5.28780   5.89893   6.02012   6.02012
 Alpha virt. eigenvalues --    6.54048  13.22233
  Beta  occ. eigenvalues --  -11.21152  -0.85454  -0.56448  -0.56448
  Beta virt. eigenvalues --    0.12205   0.15985   0.22644   0.22644   0.37981
  Beta virt. eigenvalues --    0.37981   0.53425   0.56436   0.64562   0.67312
  Beta virt. eigenvalues --    0.67312   0.78081   0.78081   1.02531   1.09584
  Beta virt. eigenvalues --    1.09584   1.11441   1.17700   1.29179   1.29179
  Beta virt. eigenvalues --    1.44736   1.44736   1.55059   1.55059   1.82763
  Beta virt. eigenvalues --    2.39940   2.46090   2.55678   2.55678   2.86375
  Beta virt. eigenvalues --    2.86375   2.94466   2.94466   3.16466   3.23171
  Beta virt. eigenvalues --    3.24320   3.24320   3.28639   3.31507   3.31507
  Beta virt. eigenvalues --    3.49862   3.54210   3.68324   3.68324   3.91659
  Beta virt. eigenvalues --    3.91659   3.96087   3.96087   4.16103   4.20986
  Beta virt. eigenvalues --    4.22030   4.27240   4.27240   4.40505   4.40505
  Beta virt. eigenvalues --    5.00063   5.17243   5.17243   5.20310   5.21285
  Beta virt. eigenvalues --    5.21285   5.30767   5.30767   5.90630   6.02453
  Beta virt. eigenvalues --    6.02453   6.54365  13.24040
          Condensed to atoms (all electrons):
               1          2          3          4
     1  C    5.137432   0.409601   0.409601   0.409601
     2  H    0.409601   0.531213  -0.031446  -0.031446
     3  H    0.409601  -0.031446   0.531213  -0.031446
     4  H    0.409601  -0.031446  -0.031446   0.531213
          Atomic-Atomic Spin Densities.
               1          2          3          4
     1  C    1.195558   0.001727   0.001727   0.001727
     2  H    0.001727  -0.077628   0.004494   0.004494
     3  H    0.001727   0.004494  -0.077628   0.004494
     4  H    0.001727   0.004494   0.004494  -0.077628
 Mulliken charges and spin densities:
               1          2
     1  C   -0.366236   1.200741
     2  H    0.122079  -0.066914
     3  H    0.122079  -0.066914
     4  H    0.122079  -0.066914
 Sum of Mulliken charges =   0.00000   1.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  C    0.000000   1.000000
 Electronic spatial extent (au):  <R**2>=             29.9122
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -7.1974   YY=             -7.1974   ZZ=             -8.9449
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              0.5825   YY=              0.5825   ZZ=             -1.1650
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.9138  ZZZ=              0.0000  XYY=              0.0000
  XXY=             -0.9138  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -14.6040 YYYY=            -14.6040 ZZZZ=            -10.4542 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -4.8680 XXZZ=             -4.5916 YYZZ=             -4.5916
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 9.643716840664D+00 E-N=-1.110367051498D+02  KE= 3.951185348325D+01
 Symmetry A1   KE= 3.647513328232D+01
 Symmetry A2   KE= 6.019830345248D-22
 Symmetry B1   KE= 1.933419791285D+00
 Symmetry B2   KE= 1.103300409646D+00
 1\1\GINC-GAMBA\SP\UCCSD(T)-FC\Gen\C1H3(2)\IRIKURA\17-May-2017\0\\# CCS
 D(T,maxcyc=100)/GEN gfinput geom=check scf=xqc\\BEB step 6: CCSD(T) fo
 r neutral ch3\\0,2\C,0,0.,-0.0000000011,0.\H,0,0.000000002,-1.08275178
 02,0.\H,0,-0.9376905476,0.5413758867,0.\H,0,0.9376905457,0.5413758901,
 0.\\Version=ES64L-G09RevD.01\HF=-39.5773216\MP2=-39.7355483\MP3=-39.75
 36874\MP4D=-39.7580237\MP4DQ=-39.7549933\PUHF=-39.5805115\PMP2-0=-39.7
 375699\PMP3-0=-39.7548434\MP4SDQ=-39.7556968\CCSD=-39.7562069\CCSD(T)=
 -39.7609459\S2=0.761878\S2-1=0.753145\S2A=0.750075\RMSD=3.852e-09\PG=D
 03H [O(C1),3C2(H1)]\\@


 SOME PHYSICISTS WOULD PREFER TO COME BACK TO THE IDEA OF AN OBJECTIVE
 REAL WORLD WHOSE SMALLEST PARTS EXIST OBJECTIVELY IN THE SAME SENSE AS
 STONES OR TREES EXIST INDEPENDENTLY OF WHETHER WE OBSERVE THEM.
 THIS, HOWEVER, IS IMPOSSIBLE.  -- WERNER HEISENBERG
 Job cpu time:       0 days  0 hours  0 minutes 29.9 seconds.
 File lengths (MBytes):  RWF=     41 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:00:02 2017.
